nvflare/poc/client/startup/sub_start.sh

```diff
@@ -33,14 +33,15 @@ stop_fl() {
     return
   fi
   pid=`cat $DIR/../pid.fl`
+  kill -SIGUSR2 $pid
   sleep 10
   kill -0 ${pid} 2> /dev/null 1>&2
   if [[ $? -ne 0 ]]; then
     echo "Process alreday terminated"
     return
   fi
   kill -9 $pid
-  rm -f $DIR/../pid.fl $DIR/../shutdown.fl $DIR/../restart.fl
+  rm -f $DIR/../pid.fl $DIR/../shutdown.fl $DIR/../restart.fl 2> /dev/null 1>&2
```

Comment:
just find out there is a typo "alreday" should be "already"

Reply:
Thanks.  Seems poc folder is lightly different from master template.